src/components/map/components/DeparturesDialogSheet.tsx

```diff
@@ -101,7 +101,7 @@ export const DeparturesDialogSheet = ({
             showOnlyFavorites={false}
             setShowOnlyFavorites={(_) => {}}
             testID="departuresContentView"
-            allowFavouriteSelection={false}
+            showFavorites={false}
```

Comment:
Minor thing. Seems like we are using Favourites word.

Reply:
We're consistently very inconsistent here 😅 When searching for "favourite" in the code, I get around 200 results, while "favorite" is around 1100. We should probably take the time to fix that some day... For now, I think we should try to stick to US English (favorite).